lib/game_runner.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "./board.rb"
+require "pry"
+
+class GameRunner
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run(output)
+    output.print empty_board
+  end
+
+  private
+
+  def empty_board
+    Board.new.stringify_board
```

Comment:
There's a little redundancy with "board" here. What do you think about renaming that method to `stringify`?

Reply:
ah, love it